MapboxNavigation/NavigationMapView.swift

```diff
@@ -251,6 +251,12 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         
         installUserCourseView()
         showsUserLocation = false
+        
+        if routeLineTracksTraversal {
+            if let firstRoute = routes?.first{
```

Comment:
This can be merged into one if statement.

Reply:
Thanks for the suggestion, I'll fix it.